lib/tokyo_api/actionkit.rb

```diff
@@ -8,8 +8,12 @@ def full_user(id)
       client.get_request("#{normalized_base_path}full_user/#{id}").body
     end
 
-    def user_path(id, phone_required)
-      "/#{normalized_base_path}user/#{id}?phone_required=#{phone_required}"
+    def user_path(id, required_fields: nil)
+      path = "/#{normalized_base_path}user/#{id}"
+      unless required_fields.nil?
+        path << "?required_fields=#{required_fields.join(',')}"
```

Comment:
Do we need to encode the fields to avoid getting an invalid URL?

Reply:
Good question! I _expect_ this to be called only with valid fields that are made up of lowercase letters and underscores. But it wouldn't hurt to pass them through `URI.escape`